enhancements/ingress/route-secret-injection-for-external-certificate-management.md

```diff
@@ -296,9 +290,30 @@ able successfully reload the certificate on the route.
   > The ingress-to-route behaviour will remain as is i.e. it will not make use of
   > the newly introduced field.
 
-- What should be the behaviour of `ValidationHostUpdate()` when using `externalCertificate`?
-  > Addressed by introducing `ValidationHostExternalCertificate()` and which will execute
-  > prior to the `ValidationHostUpdate()` function.
+### Permission requirements on `routes/custom-host`
+- `create` is required to set `.spec.tls.externalCertificate` on route creation. 
+  [AllocateHost()](https://github.com/openshift/openshift-apiserver/blob/bd2a35e58172010c658f4d8f4dff8f9f0eac187d/pkg/route/apiserver/registry/route/strategy.go#L75) enforces this check.
+- Either `create` or `update` is required on route updatation with non-empty `.spec.tls.externalCertificate`. 
+  [ValidateHostUpdate](https://github.com/openshift/openshift-apiserver/blob/bd2a35e58172010c658f4d8f4dff8f9f0eac187d/pkg/route/apiserver/registry/route/strategy.go#L96) enforces this check.
+  - Few examples:
+    - external-certificate-added-from-no-certificate
+    - external-certificate-added-from-nil-tls
+    - certificate-changed-to-external-certificate
+    - external-certificate-secret-name-changed
+    - **external-certificate-secret-name-unchanged**
```

Comment:
Can this be reformatted as real phrases instead of unit test case names?

Reply:
Sure, I've updated the statements